examples/views.py

```diff
@@ -1,13 +1,21 @@
+from django.contrib.messages.views import SuccessMessageMixin
 from django.urls import reverse_lazy
 from django.views import generic
 
-from bootstrap_modal_forms.generic import (BSModalLoginView,
-                                           BSModalCreateView,
-                                           BSModalUpdateView,
-                                           BSModalReadView,
-                                           BSModalDeleteView)
-
-from .forms import BookForm, CustomUserCreationForm, CustomAuthenticationForm
+from bootstrap_modal_forms.generic import (
+    BSModalLoginView,
+    BSModalCreateView,
+    BSModalUpdateView,
+    BSModalReadView,
+    BSModalDeleteView,
+    BSModalFormView,
```

Comment:
it is simpler to add new imports in this layout as you can just add a line ending with a comma.
otherwise (see for instance https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list )
but if you want to stick to the other layout, OK.
btw, in the `urlpatterns`, there is also a trailing comma (which is very fine IMHO)